app/src/main/java/io/gnosis/safe/ui/safe/NoSafeFragment.kt

```diff
@@ -1,28 +1,29 @@
-package io.gnosis.safe.ui.safe.overview
+package io.gnosis.safe.ui.safe
 
 import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
-import androidx.navigation.fragment.findNavController
+import androidx.fragment.app.FragmentActivity
+import androidx.navigation.Navigation
 import io.gnosis.safe.R
-import io.gnosis.safe.databinding.FragmentSafeOverviewBinding
+import io.gnosis.safe.databinding.FragmentNoSafesBinding
 import io.gnosis.safe.di.components.ViewComponent
 import io.gnosis.safe.ui.base.BaseFragment
 
-class SafeOverviewFragment : BaseFragment<FragmentSafeOverviewBinding>() {
+class NoSafeFragment : BaseFragment<FragmentNoSafesBinding>() {
 
     override fun inject(component: ViewComponent) {
         component.inject(this)
     }
 
-    override fun inflateBinding(inflater: LayoutInflater, container: ViewGroup?): FragmentSafeOverviewBinding =
-        FragmentSafeOverviewBinding.inflate(inflater, container, false)
+    override fun inflateBinding(inflater: LayoutInflater, container: ViewGroup?): FragmentNoSafesBinding =
+        FragmentNoSafesBinding.inflate(inflater, container, false)
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
-        binding.addSafeButton.setOnClickListener {
-            findNavController().navigate(SafeOverviewFragmentDirections.actionSafeOverviewFragmentToAddSafeNav())
+        binding.loadSafeButton.setOnClickListener {
+            Navigation.findNavController(activity as FragmentActivity, R.id.safe_overview_root).navigate(SafeOverviewFragmentDirections.actionSafeOverviewFragmentToAddSafeNav())
```

Comment:
Just calling `findNavController()` doesn't work?

Reply:
no, it will nav graph for current nav host. But we need main/global nav graph here